leshan-client-core/src/main/java/org/eclipse/leshan/client/object/Security.java

```diff
@@ -109,6 +109,15 @@ public static Security x509Bootstrap(String serverUri, byte[] clientCertificate,
                 clientPrivateKey.clone(), null, CertificateUsage.DOMAIN_ISSUER_CERTIFICATE.code);
     }
 
+    /**
+     * Returns a new security instance (X509) for a bootstrap server.
+     */
+    public static Security x509Bootstrap(String serverUri, byte[] clientCertificate, byte[] clientPrivateKey,
+            byte[] serverPublicKey, ULong certificateUsage) {
+        return new Security(serverUri, true, SecurityMode.X509.code, clientCertificate.clone(), serverPublicKey.clone(),
+                clientPrivateKey.clone(), null, certificateUsage);
```

Comment:
I changed the 0 to null since (fdc4a9834fbc8691c5c55ab65473f421078b663a)

Reply:
OK